app/models/audit/AuditTaskCommentTable.scala

```diff
@@ -12,7 +12,7 @@ case class AuditTaskComment(auditTaskCommentId: Int, auditTaskId: Int, missionId
                             ipAddress: String, gsvPanoramaId: Option[String], heading: Option[Double],
                             pitch: Option[Double], zoom: Option[Int], lat: Option[Double], lng: Option[Double],
                             timestamp: Timestamp, comment: String)
-case class GenericComment(commentType: String, username: String, gsvPanoramaId: Option[String], timestamp: Timestamp, comment: String)
+case class GenericComment(commentType: String, username: String, gsvPanoramaId: Option[String], timestamp: Timestamp, comment: String, heading: Option[Double], pitch: Option[Double], zoom: Option[Int], labelId: Int)
```

Comment:
I think what you want is to make the `labelId` an option type, and then instead of `0` for the audit task comments, you would use `None`. Then in the JavaScript, you would be checking of the `labelId` is `null` instead of 0.

Reply:
I have the code like this and I keep getting the error shown.. any thoughts?
![image](https://user-images.githubusercontent.com/71039842/100801738-3f6f9b80-33dd-11eb-9e6d-b3d2345a785f.png)

![image](https://user-images.githubusercontent.com/71039842/100801693-3088e900-33dd-11eb-8d5e-fb250e8d774c.png)

![image](https://user-images.githubusercontent.com/71039842/100801811-59a97980-33dd-11eb-83e6-6866e0b8caa2.png)

Reply:
Try replacing `None` with `(None: Option[Int])` in the query.